modules/cron/variables.tf

```diff
@@ -89,6 +89,22 @@ variable "memory" {
   description = "The memory limit for the job."
 }
 
+variable "task_count" {
+  type        = number
+  default     = 1
+  description = "The number of tasks to run. "
+}
+
+variable "parallelism" {
+  type        = number
+  default     = 1
+  description = "The number of parallel jobs to run. Must be <= task_count"
+  validation {
+    condition     = var.parallelism <= var.task_count
```

Comment:
we cannot reference another variable in the condition, we need to create an object, and then we can test